STEP/AST/PinTable.cs

```diff
@@ -7,20 +7,22 @@ public class PinTable : IPinTable
     private readonly HashSet<int> _analogTable = new();
     private readonly HashSet<int> _digitalTable = new();
 
-    public void RegisterPin(TypeVal type, int pinVal)
+    public void RegisterPin(PinDclNode pinDcl)
```

Comment:
explain this to me, PLZ

Reply:
Hvis vi tager imod en PinDclNode så får vi lidt mere info - f.eks. dens `SourcePosition`. Man kunne selvfølgelig også bare tilføje flere parametre, men i SymbolTable tager vi også mod nodes, så på den her måde er det consistent